hawkbit-dmf/hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpMessageHandlerService.java

```diff
@@ -324,10 +323,15 @@ private void updateAttributes(final Message message) {
         final DmfAttributeUpdate attributeUpdate = convertMessage(message, DmfAttributeUpdate.class);
         final String thingId = getStringHeaderKey(message, MessageHeaderKey.THING_ID, THING_ID_NULL);
 
+        try {
         controllerManagement.updateControllerAttributes(thingId, attributeUpdate.getAttributes(),
                 getUpdateMode(attributeUpdate));
+        } catch (final InvalidTargetAttributeException ex) {
```

Comment:
If you catch this exception here the DMF message sent via AMQP will be acknowledged and silently "ignored" instead of rejected into dead letter queue. So publisher (in this case the user of DMF API) won't get any feedback that attributes are invalid. Not sure if this is intended

Reply:
If so, are we obligated to wrap this exception into AmqpRejectAndDontRequeueException with intended message, so that the publisher is notified and also the message is rejected into dead letter queue?

Reply:
Wrapping the exception would only address this specific call to update attributes, it won't do you any good for other exceptions / other calls somewhere else in the code base. So you will still see logs polluted from different origin. In my opinion it's better to fix the root cause of the log message. I assume that `ConditionalRejectingErrorHandler` is the source of log statement you want to suppress, right?